go.mod

```diff
@@ -14,6 +14,8 @@ require (
 	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
 )
 
+replace github.com/konveyor/tackle2-addon => github.com/jortel/tackle2-addon v0.0.0-20231208142659-c1c71f08ffcd
```

Comment:
Unintentional/just for testing?

Reply:
Just for testing.  Will fix with command after https://github.com/konveyor/tackle2-addon/pull/49 is merged.